CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+## Rules
+
+Please observe the following ground-rules when making contributions:
+
+- **All** contributions must be made by opening a pull request into `main`
+- Please add a description to the pull request describing _what_ features you're
+  implementing or issues you are solving and _how_
```

Comment:
General why to implement it or link background information to understand the problem.

Reply:
:thumbsup: